hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/QueryPlanner.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hazelcast.sql.impl.calcite.opt;
+package com.hazelcast.sql.impl.calcite;
```

Comment:
You are moving everything to `jet` package, why leaving it here?

Reply:
It is still has a references from `sql.impl` package. I didn't want to make changeset too giant (it is big already), so I'd move it in the next PRs.